ios/app/src/ViewController.m

```diff
@@ -98,6 +98,10 @@ - (void)readyToClose:(NSDictionary *)data {
     [self _onJitsiMeetViewDelegateEvent:@"READY_TO_CLOSE" withData:data];
 }
 
+- (void)transcriptionChunkReceived:(NSDictionary *)data {
```

Comment:
Let's leave this commented out after testing. We don't want to run this in production builds. Same for Android.